### ISSUE TITLE
Add "Position" collection option

### ISSUE THREAD

**Issue by Explv** [Explv/Explv.github.io#10]
This option would allow the user to collect multiple positions without them being stored as a Path (Position[] / List<Position>) etc.

For example, the code output would be:

```
Position position = new Position(1, 2, 3);
Position position = new Position(4, 5, 6);
```

**Comment by ZappyScript** [comment]
Do you have this made already? 
https://github.com/Explv/Explv.github.io/blob/573bb7ff2962ae29c6b153280f01a31c2e211179/js/Position.js#L63

**Comment by Explv** [comment]
@ZappyScript no, that code is currently just used in Path.js when the user has only selected a single position on the map: https://github.com/Explv/Explv.github.io/blob/573bb7ff2962ae29c6b153280f01a31c2e211179/js/Path.js#L109

**Comment by NathanielPather** [comment]
I want to try to solve this issue but need some clarification.

A position is a single point consisting of an x, y, and z value.

Do you want a button to add positions on the map?
Should the position be represented by some sort of marker on the map?
After creating the position, there should be some output code for the position similar to this: https://i.imgur.com/qXygqki.png ?
